packages/server/test/support/helpers/e2e.coffee

```diff
@@ -57,19 +57,25 @@ replaceDurationSeconds = (str, p1, p2, p3, p4) ->
 
 replaceDurationFromReporter = (str, p1, p2, p3) ->
   ## duration='1589'
-
   p1 + _.padEnd("X", p2.length, "X") + p3
 
 replaceNodeVersion = (str, p1, p2, p3) ->
-  ## Node Version: 1.2.3
-
+  ## Node Version: 10.2.3
   return p1 + _.padEnd("X", p2.length, "X") + ' (/foo/bar/node)' + ' '.repeat(p3.length - 16)
 
 replaceDurationInTables = (str, p1, p2) ->
   ## when swapping out the duration, ensure we pad the
   ## full length of the duration so it doesn't shift content
   _.padStart("XX:XX", p1.length + p2.length)
 
+replaceParenTime = (str, p1) ->
+  ## could be (1 second) or (10 seconds)
+  ## need to account for shortest and longest
+  return _.padStart("(X second)", p1.length)
```

Comment:
this makes all the snapshots say `(X second)`, which reads a little weird, in normal English it would be `(X seconds)` since X is a variable

Reply:
Yeah, this is what I came up with.

Do you have a suggestion for writing this so that it replaces all variants below with a single replacement?

```
(2 seconds)
(1 second)
(10 seconds)
```

Reply:
ah, i see what the issue is, you can't replace `(1 second)` with `(X seconds)` without messing other stuff up. i guess you could update the regex to grab some extra whitespace from the beginning of the string, but that's beginning to feel really hacky. just doing `(X second)` is probably the best way then